crates/sui-core/src/authority.rs

```diff
@@ -1052,6 +1056,14 @@ impl AuthorityState {
                 }
             }
         }
+        task::spawn(async move {
+            loop {
+                sleep(Duration::from_secs(3600)).await;
+                send_telemetry_event().await;
+            }
+        })
+        .await
+        .expect("Should spawn a task to send telemetry event, but failed!");
```

Comment:
This does not do what you think it does and will never return, meaning this constructor will never finish. Also we 100% should have a way to disable this. We should not be enabling this inside of tests imo

Reply:
Sorry i seemed to miss seeing this. You'd need to remove lines 1065-1066